src/testing/code-debugging.md

```diff
@@ -783,9 +813,9 @@ begin/end, you can toggle the [`debugPrintBeginFrameBanner`][]
 and the [`debugPrintEndFrameBanner`][] booleans to print the
 beginning and end of the frames to the console.
 
-For example:
+**The print frame banner log for Example 1**
```

Comment:
We don't really use this style of introducing a block of code anywhere else, perhaps stick to introductory text and a colon for familiarity?

Reply:
Stet. This is consistent with the other log listings, which are behind bolded accordions.